src/lib/user-features.spec.ts

```diff
@@ -1,28 +1,26 @@
-import { getCookie } from '@guardian/libs';
 import { addCookie, removeCookie } from 'lib/cookies';
-import { fetchJson } from 'lib/utils/fetch-json';
-import type { UserFeaturesResponse } from 'types/membership';
-import type { AuthStatus } from './identity/api';
+import { isUserLoggedInOktaRefactor as isUserLoggedInOktaRefactor_ } from './identity/api';
 import {
-	getAuthStatus as getAuthStatus_,
-	isUserLoggedInOktaRefactor as isUserLoggedInOktaRefactor_,
-} from './identity/api';
-import {
-	accountDataUpdateWarning,
 	getDaysSinceLastOneOffContribution,
 	getLastOneOffContributionTimestamp,
-	getLastRecurringContributionDate,
-	getPurchaseInfo,
 	isAdFreeUser,
 	isDigitalSubscriber,
 	isPayingMember,
-	isPostAskPauseOneOffContributor,
 	isRecentOneOffContributor,
 	isRecurringContributor,
-	refresh,
 	shouldNotBeShownSupportMessaging,
 } from './user-features';
 
+const isUserLoggedInOktaRefactor =
+	isUserLoggedInOktaRefactor_ as jest.MockedFunction<
+		typeof isUserLoggedInOktaRefactor_
+	>;
+
+jest.mock('lib/identity/api', () => ({
+	isUserLoggedInOktaRefactor: jest.fn(),
+	getAuthStatus: jest.fn(),
```

Comment:
`getAuthStatus` no longer needs to be exported from  `src/lib/identity/api.ts` so we don't need to mock it any more

Reply:
do you mean the whole mock? or just getAuthStatus?

Reply:
Also this getAuthStatus is being used in `src/core/targeting/build-page-targeting.spec.ts`